controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -88,6 +88,16 @@ bool master_thread::init()
     set_server_max_connections(SOCKET_MAX_CONNECTIONS);
     set_select_timeout(SOCKETS_SELECT_TIMEOUT_MSEC);
 
+    LOG(DEBUG) << "persistent db enable=" << database.config.persistent_db;
+    if (database.config.persistent_db) {
+        LOG(DEBUG) << "loading clients from persistent db";
+        if (!database.load_persistent_db_clients()) {
+            LOG(ERROR) << "failed to load clients from persistent db";
```

Comment:
If it is an ERROR, you should return false. If it is not critical and you wish to continue on that scenario, print a WARNING.

Reply:
Also, the headline of the commit message is referring to "master_main" which is not (anymore).

Reply:
changed the ERROR log print to WARNING

Reply:
fixed the commit message